cgsmiles/read_cgsmiles.py

```diff
@@ -305,6 +315,13 @@ def read_cgsmiles(pattern):
                         prev_anchor = ref_anchor
                 # all branches added; then go back to the base anchor
                 prev_node = base_anchor
+            #================================================
+            #     bond orders for after branches            #
+            #================================================
+                if pattern[eon_b] in symbol_to_order:
+                    prev_bond_order = symbol_to_order[pattern[eon_b]]
+            elif pattern[eon_a+1] in symbol_to_order:
+                prev_bond_order = symbol_to_order[pattern[eon_a+1]]
```

Comment:
```suggestion
                #================================================
                #     bond orders for after branches            #
                #================================================
                if pattern[eon_b] in symbol_to_order:
                    prev_bond_order = symbol_to_order[pattern[eon_b]]
            elif pattern[eon_a+1] in symbol_to_order:
                prev_bond_order = symbol_to_order[pattern[eon_a+1]]
```

Reply:
It actually applies to both the if and the elif statement hence the alignment with the outer indentation

Reply:
Figured, but it looked so weird